src/libs/ReportUtils.ts

```diff
@@ -3662,7 +3663,7 @@ function shouldReportBeInOptionList({
 
     // All unread chats (even archived ones) in GSD mode will be shown. This is because GSD mode is specifically for focusing the user on the most relevant chats, primarily, the unread ones
     if (isInGSDMode) {
```

Comment:
NAB, but it would be nice to have some sort of comment that gives a hint what GSD mode is (in relation to focus mode and/or recent mode).